### DISCUSSION TITLE
Questions about timelapse command

### DISCUSSION THREAD

**Discussion by rspeir** [gtxaspec/wz_mini_hacks#778] [Q&A]
I've been unable to successfully run a timelapse on my Wyze Cam3 from an SSH connection. If I run
```
echo timelapse /media/mmc/time_lapse/tl01.mp4 2 2 | busybox nc localhost 4000
```
it just prints a new line and sits there doing nothing until I press Ctrl+C. 

From my understanding, this should have produced an mp4 video consisting of 2 frames taken 2 seconds apart, so I would expect _something_ to happen after 4 seconds. I've waited a full 5 minutes, and nothing ever happens. 

If I look at the /media/mmc/time_lapse path, I see a 0 byte tl01.mp4 file, so it's at least creating the output file....

If I try to initiate a new time lapse, I get an error message saying that there is already a timelapse in progress. If I run ```timelapse stop``` of ```timelapse close``` it also just hangs and does nothing until I Ctrl+C. I have to reboot the camera to get it to do anything different.

Am I doing something wrong? Is there some setting I need to enable that I have missed?

Thank you for your help.

**Comment by corbolais** [comment]
Hi @rspeir,
I had similar issues (with RTSP not working) running the wrong firmware. Did you flash a supported firmware version and verify it's been flashed successfully?
cheers
